src/components/album/Album.js

```diff
@@ -30,15 +31,41 @@ const AlbumTitle = styled('h3')({
 
 function Album({ album, wishListId }) {
   const [openDialog, setOpenDialog] = useState(false);
+  const [reviewsUpdated, setReviewsUpdated] = useState(false);
+  const [fullWidth, setFullWidth] = React.useState(true);
+  const [maxWidth, setMaxWidth] = React.useState('sm');
 
   const handleOpenDialog = () => {
     setOpenDialog(true);
   };
 
   const handleCloseDialog = () => {
     setOpenDialog(false);
+    // Reset the reviewsUpdated state when the dialog is closed
+    setReviewsUpdated(false);
   };
 
+  const handleMaxWidthChange = (event) => {
+    setMaxWidth(event.target.value);
+  };
+
+  const handleFullWidthChange = (event) => {
+    setFullWidth(event.target.checked);
+  };
```

Comment:
Will something be calling these functions? Do we expect to have the user do something to change the max width / full width values? It seems to me like these can just be static variables with no `useState` and no event handlers. But it's possible I'm misunderstanding what you're aiming to do here

Reply:
@akosasante sure! you are right, it's better to have it as a variable- I was just thinkimg we might want this modale have diferent sizes depending on where we call it